Sparkle/Base.lproj/Sparkle.strings

```diff
@@ -14,6 +14,22 @@
 
 "%@ %@ is currently the newest version available.\n(You are currently running version %@.)" = "%1$@ %2$@ is currently the newest version available.\n(You are currently running version %3$@.)";
 
+"No valid update information could be loaded." = "No valid update information could be loaded.";
+
+"No new update has been checked because the installation will require interaction, which has been prevented." = "No new update has been checked because the installation will require interaction, which has been prevented.";
+
+"A new update is available but cannot be installed because interaction has been prevented." = "A new update is available but cannot be installed because interaction has been prevented.";
+
+"An error occurred while running the updater. Please try again later." = "An error occurred while running the updater. Please try again later.";
+
+"An error occurred while encoding the installer parameters. Please try again later." = "An error occurred while encoding the installer parameters. Please try again later.";
```

Comment:
These strings are very specific for a rare situation. I'm afraid that's unnecessary burden for translators.

Reply:
I'll add comments to ones I think are rare/unlikely but at same time I don't want localized string usage in the project that doesn't have corresponding .strings entries.

Reply:
Actually I just won't localize the rare specific scenario ones.